geometric_controller/src/geometric_controller.cpp

```diff
@@ -147,6 +147,7 @@ void geometricCtrl::flattargetCallback(const controller_msgs::FlatTarget &msg) {
 
   targetPos_ = toEigen(msg.position);
   targetVel_ = toEigen(msg.velocity);
+  mavYaw_ = double(msg.yaw.data);
```

Comment:
This means that we always need to define yaw, which I think is not what we want. Can you add a case where we can specify ignoring yaw option and handle the case here?

Reply:
You are right. We can solve this by using a mask (IGNORE_YAW) in the message definition. What do you think?

Reply:
That sounds great